api/db/changelog/db.changelog-201-demographic-survey-v2-aian.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <changeSet author="dmohs" id="changelog-201-demographic-survey-v2-aian">
+    <sql>
+      ALTER TABLE demographic_survey_v2_ethnic_category
+        MODIFY COLUMN ethnic_category
```

Comment:
curious, i know this functionality is only on test however does the alter table modify enums automatically take care of existing entries for user who have selected AI_AN_AI_AN?